vpr/src/place/place.cpp

```diff
@@ -111,7 +120,7 @@ static vtr::vector<ClusterNetId, char> bb_updated_before;
  * blocks on each of a net's bounding box (to allow efficient updates),      *
  * respectively.                                                             */
 
-static vtr::vector<ClusterNetId, t_bb> bb_coords, bb_num_on_edges;
+vtr::vector<ClusterNetId, t_bb> bb_coords, bb_num_on_edges;
```

Comment:
This is now a full global variable.  It should go in placer_context.h, probably as a new PlaceMoveContext class. Other global variables like init_rlim should also go in that class if you need them to be global.

Reply:
Done.